tslint.json

```diff
@@ -1,23 +1,48 @@
 {
+  "rulesDirectory": [
+    "node_modules/codelyzer"
+  ],
   "rules": {
-    "class-name": true,
-    "comment-format": [true, "check-space"],
-    "curly": true,
```

Comment:
Sorry it's a mistake to commit tslint.json

Envoyé de mon iPhone

> Le 26 avr. 2016 à 18:10, Grégory Copin <notifications@github.com> a écrit :
> 
> In tslint.json:
> 
> >    "rules": {
> > -    "class-name": true,
> > -    "comment-format": [true, "check-space"],
> > -    "curly": true,
> why changing tsling rules ? Current code is based on those rules
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
>